pg_stream_copy/protocol.py

```diff
@@ -1,12 +1,14 @@
-from datetime import date
+from datetime import date, datetime
+from time import mktime
 from decimal import Decimal
 from struct import pack
-from typing import List, Tuple, cast
+from typing import List, Tuple, cast, Union
 
 # https://www.postgresql.org/docs/10/sql-copy.html - Binary Format section
 
 pg_null = pack('>I', 0xFFFFFFFF)
 pg_date_epoch = date(2000, 1, 1)  # https://www.postgresql.org/message-id/d34l6e%24284h%241%40news.hub.org
+pg_date_epoch_ts = int(mktime(pg_date_epoch.timetuple()))
```

Comment:
pg_datetime_epoch `datetime.utcfromtimestamp(0)` ?

Reply:
I think your solution here is fine, as I cant think of any better method for calculating timestamps diffs.
The only thing -  would call it `pg_timestamp_epoch`